sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -109,6 +112,16 @@ class ProjectOverviewViewModel {
         projectOverviews.remove(projectOverview)
     }
 
+    /**
+     * Stores the subscription checkbox with its associated projectcode in the  
+     projectToCheckbox map. 
+     * @param code projectCode associated with a ProjectSummary
+     * @param checkbox subscription checkbox of a ProjectSummary
+     */
+    void addSubscriptionCheckbox(String code, CheckBox checkBox){
+        projectToCheckbox.put(code, checkBox)
+    }
```

Comment:
What happens when this method it called twice for the same project code?

Reply:
its a map, there will be not duplicate